File: Coding_Dojo/WebFundamentals/Week 1/Day 5/Practice/loopchallenger/loop.js
//Using a loop write code that will console.log all of the odd values from 1 up to 20.
function log(){
    for(var i=1;i<=20;i++){
      console.log(i)

    }
}
log();

//Using a loop write code that will console.log all of the values that are evenly divisible by 3 from 100 down to 0.
function div3(){
    for(var i=100;i=>0;i--){
     if (i%3==0){
        console.log(i);
     }

    }
}

// Using a loop write code that will console.log the values in this sequence 4, 2.5, 1, -0.5, -2, -3.5.
function seq(){
    for (var i=4;i>=-3,5;i-=1.5){
        console.log(i)

}

}

// Write code that will add all of the values from 1-100 onto some variable sum and at the end console.log the result 1 + 2 + 3 + ... + 98 + 99 + 100. We should get back 5050 at the end.
 function some(){
  var some=0;
    for (var i=1;i<101;i++){
   som+=i;

  }

 }
 // Write code that will multiply all of the values from 1-12 onto some variable product and at the end console.log the result 1 * 2 * 3 * ... * 10 * 11 * 12. We should get back 479001600 at the end.
 function mult(){
    var some=1;
      for (var i=1;i<13;i++){
     som*=i;
      }
    }

File: Coding_Dojo/Python/Week 1/Day 1/Core/loop.py
# Basic - Print all integers from 0 to 150.


for i in range (0,159,1):
    print(i)


#Multiples of Five - Print all the multiples of 5 from 5 to 1,000


for index in range (5,1001,1):
    if (index%5==0):
        print(index)




#Counting, the Dojo Way - Print integers 1 to 100. If divisible by 5, print "Coding" instead. If divisible by 10, print "Coding Dojo".

for codo in range(1, 101, 1):
    if codo % 10 == 0:
        print("Coding Dojo")
    elif codo % 5 == 0:
        print("Coding")
    else:
        print(codo)



# Whoa. That Sucker's Huge - Add odd integers from 0 to 500,000, and print the final sum.


sum=0
for x in range(0,500001):
    sum+=x
print(sum)





# Countdown by Fours - Print positive numbers starting at 2018, counting down by fours.


for j in range (2018,0,-4):
    print(j)




# Flexible Counter - Set three variables: lowNum, highNum, mult. Starting at lowNum and going through highNum, print only the integers that are a multiple of mult. For example, if lowNum=2, highNum=9, and mult=3, the loop should print 3, 6, 9 (on successive lines)
lowNum=2
highNum=9
mult=3


for i in range (lowNum,highNum+1):
    if i%mult==0 :
        print(i)